src/components/NotificationBar/NotificationBar.jsx

```diff
@@ -12,6 +12,12 @@ export default class NotificationBar extends React.Component {
     return (
       <div className={ `notification-bar ${dismissedMod}` }>
         <Container className="notification-bar__inner">
+          <p>
```

Comment:
Heh, since webpack v4 just went release, would you mind linking to this medium article instead, and then keep the other links. [This](https://medium.com/webpack/webpack-4-released-today-6cdb994702d4) is the new article, and you can link this instead.

Reply:
Done

Reply:
Done